### PR TITLE
Fix integration cli

### PR DESCRIPTION
**Pull request by icecrime**
Quick & dirty, but we need green tests!

### CODE REVIEW

**Comment by jfrazelle** [comment]
LGTM

**Comment by calavera** [comment]
LGTM

File: integration-cli/docker_cli_run_test.go [hunk 1]
```diff
@@ -2754,19 +2754,24 @@ func (s *DockerSuite) TestRunUnshareProc(c *check.C) {
 	// Not applicable on Windows as uses Unix specific functionality
 	testRequires(c, Apparmor, NativeExecDriver, DaemonIsLinux)
 
+	// Different graphdriver yield different error messages.
+	isPermissionErr := func(s string) bool {
+		return strings.Contains(s, "Permission denied") || strings.Contains(s, "Operation not permitted")
```
Comment (LK4D4) [inline integration-cli/docker_cli_run_test.go]: let's `ToLower(s)` and use undercase for samples. or `ToUpper` CAUSEWELIKECAPS.

Comment (calavera) [inline integration-cli/docker_cli_run_test.go]: :laughing: :LAUGHING:

**Comment by icecrime** [comment]
I surrender and skipped both tests: I fail to understand why `TestRunUnshareProc` fails randomly.

**Comment by duglin** [comment]
https://www.youtube.com/watch?v=ywDZKfZ-iZA  :-)

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**